src/main/java/seedu/address/logic/parser/ListCommandParser.java

```diff
@@ -24,6 +24,9 @@ public ListCommand parse(String args) throws ParseException {
 
         if (argMultimap.getValue(PREFIX_SORT).isPresent()) {
             sortField = argMultimap.getValue(PREFIX_SORT).get();
+            if (!sortField.equals("name") && !sortField.equals("email")) {
+                throw new ParseException(ListCommand.MESSAGE_INVALID_SORT_FIELD);
```

Comment:
Test case for when the sort field is invalid should be added.

Reply:
Thanks for pointing out! Added these